pkg/examples/typescript/yarnworkspacespostinstall/pkg1/src/index.ts

```diff
@@ -0,0 +1 @@
+export const name = "1"
```

Comment:
You'll want to confirm that the `postinstall` hook is actually running. Put something unique here and then add it as a check above.

```suggestion
export const name = "Look! Up in the sky! It's a bird! It's a plane! It's Superman!"
```

Reply:
Fixed.